.php_cs.dist

```diff
@@ -0,0 +1,40 @@
+<?php
+
+$config = PhpCsFixer\Config::create()
+    ->setRiskyAllowed(true)
+    ->setRules([
+        '@PSR2' => true,
+        'array_syntax' => ['syntax' => 'short'],
```

Comment:
Can we sort these from PSR-2 and below..?

There is a Phpstorm plugin if you wish..

Reply:
Copy-pasted from Guzzle and I want to keep in sync with it.